app/models/competency.rb

```diff
@@ -2,4 +2,6 @@ class Competency < ApplicationRecord
   belongs_to :work_process
 
   validates :sort_order, uniqueness: {scope: :work_process}
+
+  has_many :competency_options, as: :resource
```

Comment:
Just for organization's sake, could we move the `has_many` statement up so the `belongs_to` and `has_many` are all grouped together and not broken up by the `validates statement?